packages/mdc-textfield/foundation.js

```diff
@@ -79,13 +99,40 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.adapter_.deregisterInputBlurHandler(this.inputBlurHandler_);
     this.adapter_.deregisterInputInputHandler(this.inputInputHandler_);
     this.adapter_.deregisterInputKeydownHandler(this.inputKeydownHandler_);
+    this.adapter_.deregisterTextFieldClickHandler(this.textFieldClickHandler_);
+  }
+
+  handleTextFieldClick_(evt) {
+    const {target} = evt;
+    const {TEXT_FIELD_ICON} = MDCTextfieldFoundation.cssClasses;
+
+    if (this.adapter_.eventTargetHasClass(target, TEXT_FIELD_ICON)) {
```

Comment:
Maybe we should just log a future issue for this, but it occurs to me that it's weird that we specifically only emit icon action events when the icon has a tab stop...but it's impossible to trigger this from keyboard input.

Reply:
Hmmm... I think it should. I think I'll add it.